Product/common.css

```diff
@@ -4,21 +4,21 @@ body
     box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
     font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
 }
-a > .fa 
+:root 
+{   
+    --drop-hover-color: #ddd;    
```

Comment:
People told me to use uppercae for the variables so just passing down that suggestion , it;s just a preference thing.

Reply:
ohh okay. I will check. But for now, keeping it like this only. Thanks for the suggestion